consensus/consensus-types/src/quorum_cert.rs

```diff
@@ -76,7 +76,7 @@ impl QuorumCert {
         genesis_id: HashValue,
     ) -> QuorumCert {
         let ancestor = BlockInfo::new(
-            ledger_info.epoch() + 1,
+            ledger_info.epoch().saturating_add(1),
```

Comment:
we probably want to panic here, but also make sure that this cannot be exploited :D